interfaces/builtin/alsa.go

```diff
@@ -1,7 +1,7 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
 
 /*
- * Copyright (C) 2016 Canonical Ltd
+ * Copyright (C) 2017 Canonical Ltd
```

Comment:
No need to update any of these copyright years. The year is supposed to indicate the first time this file was published publicly.